rts/Game/Camera/SpringController.cpp

```diff
@@ -30,10 +31,30 @@ CSpringController::CSpringController()
 	, maxDist(std::max(mapDims.mapx, mapDims.mapy) * SQUARE_SIZE * 1.333f)
 	, oldDist(0.0f)
 	, zoomBack(false)
-	, cursorZoomIn(configHandler->GetBool("CamSpringZoomInToMousePos"))
-	, cursorZoomOut(configHandler->GetBool("CamSpringZoomOutFromMousePos"))
 {
 	enabled = configHandler->GetBool("CamSpringEnabled");
+	configHandler->NotifyOnChange(this, {"CamSpringScrollSpeed", "CamSpringFOV", "CamSpringZoomInToMousePos", "CamSpringZoomOutFromMousePos", "CamSpringFastScale"});
+	ConfigUpdate();
+}
+
+CSpringController::~CSpringController()
+{
+	configHandler->RemoveObserver(this);
+}
+
+
+void CSpringController::ConfigUpdate()
+{
+	scrollSpeed = configHandler->GetFloat("CamSpringScrollSpeed") * 0.1f;
```

Comment:
Now we can remove `scrollSpeed` and `fov` assignment on `Update`

Reply:
I replaced some more config get lines that seemed like they might impact performance since I was already doing this kind of changes.